govcd/api_vcd_test.go

```diff
@@ -90,13 +92,16 @@ type TestConfig struct {
 			SP1 string `yaml:"storageProfile1"`
 			SP2 string `yaml:"storageProfile2,omitempty"`
 		} `yaml:"storageProfile"`
-		ExternalIp      string `yaml:"externalIp,omitempty"`
-		ExternalNetmask string `yaml:"externalNetmask,omitempty"`
-		InternalIp      string `yaml:"internalIp,omitempty"`
-		InternalNetmask string `yaml:"internalNetmask,omitempty"`
-		EdgeGateway     string `yaml:"edgeGateway,omitempty"`
-		ExternalNetwork string `yaml:"externalNetwork,omitempty"`
-		Disk            struct {
+		ExternalIp                   string `yaml:"externalIp,omitempty"`
+		ExternalNetmask              string `yaml:"externalNetmask,omitempty"`
+		InternalIp                   string `yaml:"internalIp,omitempty"`
+		InternalNetmask              string `yaml:"internalNetmask,omitempty"`
+		EdgeGateway                  string `yaml:"edgeGateway,omitempty"`
+		ExternalNetwork              string `yaml:"externalNetwork,omitempty"`
+		ExternalNetworkPortGroup     string `yaml:"externalNetworkPortGroup,omitempty"`
+		ExternalNetworkPortGroupType string `yaml:"externalNetworkPortGroupType,omitempty"`
+		VimServer                    string `yaml:"vimServer,omitempty"`
```

Comment:
Can we call this field `Vcenter`, as it is in terraform-provider-vcd?

Reply:
Yes, I can. If everyone agrees on that. My idea was to match more vCD api in govcd, in terraform vcd GUI.

Reply:
It's better if we match vCD API naming in go-vcloud-director - govcd should provide a relatively thin and easily translatable layer over the vCD API.